src/Sylius/Bundle/ApiBundle/Resources/config/api_platform/resources/admin/AdminUserPassword.xml

```diff
@@ -48,11 +48,6 @@
                         </value>
                     </values>
                 </validationContext>
-                <openapiContext>
```

Comment:
We need to cover these removed OpenApi contexts in different way, but we could add it separately 🖖🏻

Reply:
OpenapiContext is not available in the schema in 4.0

Reply:
The `openapiContext` is now accessible under the name `openapi`:
https://github.com/api-platform/core/blob/3.4/src/OpenApi/Factory/OpenApiFactory.php#L233